demo/src/main.css

```diff
@@ -134,11 +134,14 @@ footer {
     margin-top: 10px;
 }
 .code__button {
-    font-weight: bold;
     cursor: pointer;
     display: inline-block;
     padding: 10px 0;
     color: var(--colorPageLinks);
+    background-color: transparent;
+    font: 16px/1.5 sans-serif;
+    font-weight: bold;
```

Comment:
AI guess there's a question of how styled this should be. Are our current buttons bold? https://react-accessible-accordion.springload.co.nz/

Reply:
This is the Show code button css, yep its bold, I just moved the rule down to be with the other text ones